cfgov/jinja2/v1/the-bureau/bureau-structure/_vars-division-data.html

```diff
@@ -0,0 +1,524 @@
+        '_id': 'consumer-engagement',
+        'order': 10
+    }, {
+        'relative_url': '/orgmember/consumer-education-engagement/financial-education/',
+        'titles': ['Assistant Director'],
+        'id': 36727,
+        'category': 'Divisions',
+        'status': 'publish',
+        'parent': 36723,
+        'slug': 'financial-education',
+        'comment_status': 'closed',
+        'title_plain': 'Financial Education',
+        'name': 'Janneke Ratcliffe',
+        'url': 'http://www.consumerfinance.gov/orgmember/consumer-education-engagement/financial-education/',
+        'title': 'Financial Education',
+        '_id': 'financial-education',
+        'order': 20
+    }, {
+        'relative_url': '/orgmember/consumer-education-engagement/financial-empowerment/',
+        'titles': ['Assistant Director'],
+        'id': 36729,
+        'category': 'Divisions',
+        'status': 'publish',
+        'parent': 36723,
+        'slug': 'financial-empowerment',
+        'comment_status': 'closed',
+        'title_plain': 'Financial Empowerment',
+        'name': 'Daniel Dodd-Ramirez',
+        'url': 'http://www.consumerfinance.gov/orgmember/consumer-education-engagement/financial-empowerment/',
+        'title': 'Financial Empowerment',
+        '_id': 'financial-empowerment',
+        'order': 30
+    }, {
+        'relative_url': '/orgmember/consumer-education-engagement/older-americans/',
+        'titles': ['Assistant Director'],
+        'id': 36731,
+        'category': 'Divisions',
+        'status': 'publish',
+        'parent': 36723,
+        'slug': 'older-americans',
+        'title_plain': 'Older Americans',
+        'name': 'Nora Dodd Eisenhower',
+        'url': 'http://www.consumerfinance.gov/orgmember/consumer-education-engagement/older-americans/',
+        'title': 'Older Americans',
+        '_id': 'older-americans',
+        'order': 40
+    }, {
+        'relative_url': '/orgmember/consumer-education-engagement/servicemember-affairs/',
+        'titles': ['Assistant Director'],
+        'id': 36733,
+        'category': 'Divisions',
+        'status': 'publish',
+        'parent': 36723,
+        'slug': 'servicemember-affairs',
+        'title_plain': 'Servicemember Affairs',
+        'name': 'Hollister Petraeus',
+        'url': 'http://www.consumerfinance.gov/orgmember/consumer-education-engagement/servicemember-affairs/',
+        'title': 'Servicemember Affairs',
+        '_id': 'servicemember-affairs',
+        'order': 50
+    }, {
+        'relative_url': '/orgmember/consumer-education-engagement/students/',
+        'excerpt': '',
+        'titles': ['Assistant Director'],
+        'id': 36735,
+        'category': 'Divisions',
+        'status': 'publish',
+        'parent': 36723,
+        'slug': 'students',
+        'title_plain': 'Students',
+        'name': 'Seth Frotman',
+        'url': 'http://www.consumerfinance.gov/orgmember/consumer-education-engagement/students/',
```

Comment:
Do we need `http://www.consumerfinance.gov` on these `url`s? When it's live it shouldn't need it, right? Also it could be added at the point where the data is loaded in.